docs/Data-Loading-Maintaining-Studies.md

```diff
@@ -0,0 +1,39 @@
+# Data loading: Importing without validation and deleting studies
+For data curators and developers cbioportalImporter.py is available. This script can import data regardless of validation results. If data format is incorrect, the importer may stop with an error or crash, or leave the database in an inconsistent state.
```

Comment:
One thing I think would make this better is linking to the tools you'd suggest for people who ended up here (by browsing the `docs/` folder or using the GitHub text search) but *don't* fall in this niche: <https://github.com/cBioPortal/cbioportal/blob/v1.16.0/docs/Data-Loading.md> (but using a relative link of course).